Eshop.Persistence/EshopDbContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Eshop.Domain;
+using Microsoft.EntityFrameworkCore;
+
+namespace Eshop.Persistence
+{
+    public class EshopDbContext : DbContext
+    {
+#pragma warning disable CS8618
+        public EshopDbContext(DbContextOptions<EshopDbContext> options) : base(options) 
+        {
+#pragma warning restore CS8618
+        }
+
+        public DbSet<Category> Categories  { get; private set; }
```

Comment:
remind me next session that we have to remove private set;
but with this we need to upgrade EF from 6.0.33 -> 6.4.4

Reply:
okay